pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -257,6 +259,63 @@ func TestReconcilerReconcileUsingVolumes(t *testing.T) {
 	})
 }
 
+func TestServiceNotFound(t *testing.T) {
+	backingServiceResourceRef := "backingService1"
+	matchLabels := map[string]string{
+		"connects-to": "database",
+		"environment": "reconciler",
+	}
+	f := mocks.NewFake(t, reconcilerNs)
+	f.AddMockedUnstructuredServiceBinding(reconcilerName, backingServiceResourceRef, "", deploymentsGVR, matchLabels)
+	f.AddMockedUnstructuredCSV("cluster-service-version-list")
```

Comment:
CSV might not be necessary
depends whether we are checking via annotations on CRDs or via x-descriptors